### DISCUSSION TITLE
Connection timeout error for zynq board & OSError for kv260 board

### DISCUSSION THREAD

**Discussion by Raymonna** [bol-edu/HLS-SOC-Discussions#65] [Q&A] [answered]
Hi all, 
I currently encounter the connection timeout problem in zynq board(I have tried zynq_01, zynq_06 and zyn_11) and OSError in kv260 board(have tried kv260_01, kv260_06)
When I double check the block design, it looks as follows,

![lab2_block_design](https://github.com/bol-edu/HLS-SOC-Discussions/assets/24355578/c9b88fde-105f-421e-994c-3022a82072a8)

It should be completely the same as the workbook_lab2 shown. Is there any other step that could cause this error?

Thank you in advance ><

**Comment by tim21525** [comment, marked as answer]
It works if I use pynq_17.
You can try it

**Reply by Raymonna** [reply]
Thank you for your quick reply.
Although it still not works for me, but when I use kv260_07 and change the 
ol = Overlay("/home/xilinx/jupyter_notebooks/FIRN11MAXI.bit") into 
ol = Overlay("/home/root/jupyter_notebooks/FIRN11MAXI.bit"), 
it finally generates the desired output for me!

Again, thank you for your responding!
